### PR TITLE
0.0.26

### PR DESCRIPTION
**Pull request by deepunbxd005**

### CODE REVIEW

**Review by sanjithkumar017 — APPROVED** [review]

File: demo/src/components/CombinedFilters.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ const CombinedFilters = () => {
     return (
         <CombinedFacets
             transform={transform}
-            RangeFacetItemComponent={RangeFacetItemComponent}
-            TextFacetItemComponent={TextFacetItemComponent}
+            RangeFacetItemComponent={<RangeFacetItemComponent />}
```
Comment (sanjithkumar017) [inline demo/src/components/CombinedFilters.js:16]: rename the prop to denote that the value is an instance. `RangeFacetItemComponent` -> `rangeFacetItemComponent `

File: demo/src/components/CombinedFilters.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ const CombinedFilters = () => {
...
+            TextFacetItemComponent={<TextFacetItemComponent />}
```
Comment (sanjithkumar017) [inline demo/src/components/CombinedFilters.js:17]: rename the prop to denote that the value is an instance. `TextFacetItemComponent ` -> `textFacetItemComponent `

Comment (sanjithkumar017) [inline demo/src/components/CombinedFilters.js:17]: So we were accepting components before. Now that we are planning to make a switch to instances. Should we be also changing the way we are handling the instance.

File: demo/src/components/CombinedFilters.js (hunk 1 above)
Comment (deepunbxd005) [reply demo/src/components/CombinedFilters.js]: done

File: demo/src/components/CombinedFilters.js (hunk 2 above)
Comment (deepunbxd005) [reply demo/src/components/CombinedFilters.js]: done

Comment (deepunbxd005) [reply demo/src/components/CombinedFilters.js]: done

**Review by sanjithkumar017 — APPROVED** [review]

**Pull request merged**